Add note about using timeformat for partition templates (#5465)
influxdata/docs-v2: content/influxdb/cloud-dedicated/reference/cli/influxctl/table/create.md

Each of these can be used as part of the partition template.
Be sure to follow [partitioning best practices](/influxdb/cloud-dedicated/admin/custom-partitions/best-practices/).

{{% note %}}
#### Always provide a time format when using custom partitioning

If defining a custom partition template for your table with any of the
`--template-*` flags, always include the `--template-timeformat` flag with a

> time format to use in your partition template. Otherwise time will be omitted
> from the partition template and partitions won't be able to be compacted.

Reviewer: ```suggestion
time format to use in your partition template.
Otherwise, InfluxDB omits time from the partition template and won't compact partitions.
```
